postfix/README.md

```diff
@@ -13,7 +13,22 @@ Install the `dd-check-postfix` package manually or with your favorite configurat
 
 ## Configuration
 
-Edit the `postfix.yaml` file to point to your server and port, set the masters to monitor
+* Edit the `postfix.yaml` file to point to your postfix spool/queues directory (e.g. `/var/spool/postfix)..
+* Add a few mail queues to monitor (e.g. `incoming`, `active`, and `deferred`).
+* Update `/etc/sudoers` to allow `dd-agent` to run `find /path/to/postfix/spool/* -type f`.
```

Comment:
I see this PR adds an example sudoers entry in `check.py`, but we should probably add it here, too.

Also be good to be clear that `dd-agent` is a user, i.e. "to allow the user `dd-agent` to run"